kenjutsu/kenjutsu.py

```diff
@@ -285,6 +295,11 @@ def len_slice(a_slice, a_length=None):
             4
     """
 
+    if isinstance(a_slice, numbers.Integral):
+        raise TypeError(
+            "An integral index does not provide an object with a length."
+        )
```

Comment:
May revisit this in a future PR. For now am going to leave this exception alone.